sources/examples/README.rst

```diff
@@ -0,0 +1,39 @@
+OpenDylan example projects
+==========================
+
+Each sub-directory contains a standalone Dylan project. To compile them, you should have 
+a `working OpenDylan <https://opendylan.org/download/index.html>`_ installation. 
```

Comment:
Open Dylan is generally written with a space.  (Here and in the title above.)

Reply:
Fixed